framework/UI/DendrogramView.py

```diff
@@ -411,18 +411,19 @@ def selectionChanged(self):
       @ In, None
       @ Out, None
     """
-    ## Disable the communication so we don't end up in infinite callbacks
-    self.scene().selectionChanged.disconnect(self.select)
+    # ## Disable the communication so we don't end up in infinite callbacks
+    # self.scene().selectionChanged.disconnect(self.select)
```

Comment:
I would like to keep it in, as this feature is not yet supported by us, but it may be added in the future, and I think this is a reasonable start/hint of how to proceed. The feature is the selection of nodes in a hierarchical tree view so that users can see in a corresponding scatter plot which points are associated to one or a group of clusters.